### ISSUE TITLE
in side nav item, the icon's title is overlapping with the tooltip

### ISSUE THREAD

**Issue by zuixinwang** [nuxt/devtools#198]
the TabIcon.vue component is used in 3 places, should I fix it only in the side nav?
If yes or please tell me how to adjust, I will fix later. Thanks.
@antfu

**Comment by antfu** [comment]
What do you mean? Any reproduction steps and screenshots?

**Comment by zuixinwang** [comment]
> What do you mean? Any reproduction steps and screenshots?

sorry for my unclear expression.
here is the photo

![20230424091829](https://user-images.githubusercontent.com/59717852/233878916-c45ca609-a5b4-49cb-9308-162b60f41295.jpg)

the TabIcon.vue component is used in 3 places, should I fix it **only** in the side nav?

**Comment by antfu** [comment]
Yeah it seems the tooltip is only overlaps on sidenav, Pr welcome to fix it.

**Comment by zuixinwang** [comment]
I committed a PR
there may be a better implement
